helm/draughtsman-chart/templates/deployment.yaml

```diff
@@ -45,6 +45,12 @@ spec:
             port: 8000
           initialDelaySeconds: 15
           timeoutSeconds: 1
+        lifecycle:
+          preStop:
+            exec:
+              command:
+              - sleep
+              - "5"
```

Comment:
This needs to be added because if we are updating draughtsman itself then pod will be killed before it can set the github deployment event to "successful" and this will create an infinite loop.

fix is  have proper handling of term signal in draughtsmen but that is out of scope fo this PR

Reply:
issue for tracking https://github.com/giantswarm/giantswarm/issues/2549